features/step_definitions/basic_steps.rb

```diff
@@ -30,3 +30,7 @@
 Then(/^show me the page$/) do
   save_and_open_page
 end
+
+Then(/^I should see "([^"]*)" link$/) do |link|
+  find_link(link).visible?
```

Comment:
I'm not sure if this is a proper assertion. Personally I always use the `expect` keyword. @diraulo @AmberWilkie any opinion on this?